libs/stocks/feature-shell/src/lib/stocks/stocks.component.html

```diff
@@ -16,18 +16,72 @@
       >
     </mat-error>
   </mat-form-field>
+  <mat-form-field>
+    <mat-label>From date:</mat-label>
+    <input
+      [class.error-field]="shouldDisplayDateError"
+      matInput
+      formControlName="from"
+      [min]="minDate"
+      [max]="maxDate"
+      [matDatepicker]="from"
+      placeholder="Choose a date"
+    />
+    <mat-error>
+      <span
+        *ngIf="
+          !stockPickerForm.get('from').valid &&
```

Comment:
This complicated of a conditional in the HTML could go in a getter in the component